src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public static void getRandomColor() {
+        Random random = new Random();
+        int index = new Random().nextInt(Color.values().length);
+    }

```

Comment:
The method getRandomColor() should not be static according to the checklist. Instead of using a static method, you should create an instance of ColorSupplier and call the method on that instance.